Core/Services/StitchCoreRemoteMongoDBService/Sources/StitchCoreRemoteMongoDBService/Sync/Internal/DocumentVersionInfo.swift

```diff
@@ -64,7 +64,7 @@ final class DocumentVersionInfo {
         /// the GUID instance id of this version
         let instanceId: String
         /// the version counter of this version
-        let versionCounter: Int64
+        let versionCounter: Int
```

Comment:
[minor] I think this needs to be an Int64. (probably super unlikely, but If this ever got run on a 32-bit system, this would not behave correctly in the case that a document is updated more than 2 billion times)

Reply:
ditto